File: test_examples/dazzer/target.py
# target.py
import json

def parse_json(data):
    try:
        result = json.loads(data)
        return True
    except json.JSONDecodeError:
        return False

# Функция-мишень для Dazzer
def target(input_data):
    return parse_json(input_data)

File: test_examples/go-fuzz/target.go
// target.go
package json

import "encoding/json"

// Функция-мишень для Go-fuzz
func Fuzz(data []byte) int {
    var j interface{}
    if err := json.Unmarshal(data, &j); err != nil {
        return 0
    }
    return 1
}
